basic.man

```diff
@@ -1052,6 +1061,48 @@ th_plan$                  - returns the user's .plan as a string
     <3
    
 
+ TH_TIME
+ -------
+ Returns the current UNIX timestamp
+
+
+ TH_LOCALTIME[$]
```

Comment:
what is [$] for?

Reply:
The [$] denotes that the sigil is optional.  With the `$` the functions return a string of the timestamp, and without it they will return integers based on the input option:
```
>print th_localtime$(0)
Thursday, January 1, 1970 6:00:00 AM +06
```

Reply:
The arguments are also optional, so these will return the current unix time if no args are supplied:
```
>print th_localtime$
Thursday, December 10, 2020 11:06:53 PM +06
>print th_gmtime$
Thursday, December 10, 2020 5:07:16 PM UTC
```
Note that the time zone of TH_LOCALTIME[$] is whatever the user has set in `set tz`.